yaus/apps/admin/src/app/app.tsx

```diff
@@ -1,14 +1,52 @@
-import * as React from "react";
-import { Admin, Resource } from "react-admin";
-import jsonServerProvider from "ra-data-json-server";
-import { PostList } from "./posts";
-
-const App = () => (
-  <Admin
-    dataProvider={jsonServerProvider("https://jsonplaceholder.typicode.com")}
-  >
-    <Resource name="posts" list={PostList} options={{}} />
-  </Admin>
-);
-
-export default App;
\ No newline at end of file
+import * as React from 'react';
+import { Admin, Resource, DataProvider, ListGuesser } from 'react-admin';
+import buildHasuraProvider from 'ra-data-hasura';
```

Comment:
`ra-data-hasura` may not work since we are making some changes and validateion whe adding URLs. For now it will work but we will need to make some changes.